nnvm/tests/python/frontend/darknet/test_forward.py

```diff
@@ -66,14 +68,16 @@ def _get_tvm_output(net, data):
     m.set_input(**params)
     m.run()
     # get outputs
-    out_shape = (net.outputs,)
-    tvm_out = m.get_output(0, tvm.nd.empty(out_shape, dtype)).asnumpy()
+    tvm_out = m.get_output(0).asnumpy().flatten()
```

Comment:
if you flatten, the output shape cannot be validated.

Reply:
Done